### PR TITLE
Change confirm prompt for relationships management

### PR DESCRIPTION
**Pull request by tribela**
On relationships manager, Just asking same "Are you sure?" for destructive action is not good.
Especially, for mobile interface, User cannot know if they touched different action

It maybe better if "Are you sure to ~~(action) for X users?", But it is fine with confirming with action like "Are you sure to follow selected followers?"

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

**Review by antimundo — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -1322,6 +1322,9 @@ en:
       unrecognized_emoji: is not a recognized emoji
   relationships:
     activity: Account activity
+    confirm_follow_selected_followers: Are you sure to follow selected followers?
```
Comment (TobyWilkes) [inline config/locales/en.yml:1325]:
```suggestion
    confirm_follow_selected_followers: Are you sure you want to follow selected followers?
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -1322,6 +1322,9 @@ en:
...
+    confirm_remove_selected_followers: Are you sure to remove selected followers?
```
Comment (TobyWilkes) [inline config/locales/en.yml:1326]:
```suggestion
    confirm_remove_selected_followers: Are you sure you want to remove selected followers?
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -1322,6 +1322,9 @@ en:
...
+    confirm_remove_selected_follows: Are you sure to remove selected follows?
```
Comment (TobyWilkes) [inline config/locales/en.yml:1327]:
```suggestion
    confirm_remove_selected_follows: Are you sure you want to remove selected follows?
```

**Review by TobyWilkes — CHANGES_REQUESTED** [review]
PR is currently marked as approved, but the en yml has syntax issues

**Review by ClearlyClaire — APPROVED** [review]

**Comment by ClearlyClaire** [comment]
@tribela can you rebase it on latest main?

**Pull request merged**